lib/capabilities.js

```diff
@@ -93,45 +65,21 @@ var capabilities = {
     checks: [
       {
         check: function(cb) {
-          if (license('agent:metrics')) {
-            cb(true);
-          } else {
-            cb(false, 'Metrics license is missing or expired.');
-          }
-        }
-      },
-      {
-        check: function(cb) {
           if (agent().metrics) {
             cb(true);
           } else {
             cb(false, 'Metrics is disabled.');
           }
         }
       },
-      {
-        check: function(cb) {
-          var addon = require('strong-agent/lib/addon');
-          if (addon) {
-            cb(true);
-          } else {
-            cb(false, 'Strong-agent addon not compiled.');
-          }
-        }
-      }
     ]
   },
   cpuprofile: {
     name: 'CPU profiling',
     checks: [
       {
         check: function(cb) {
-          var addon = require('strong-agent/lib/addon');
-          if (addon) {
-            cb(true);
-          } else {
-            cb(false, 'Strong-agent addon not compiled.');
-          }
+          cb(true);
```

Comment:
I'm not sure if it would be clearer or not, but making the `checks` array empty would have the same result but with less overhead (not that it's measurable here).